README.md

```diff
@@ -25,7 +25,34 @@ Here's a oneliner on how I installed everything:
 ```$ sudo pacman -Syu && sudo pacman -S i3-gaps feh i3-dmenu-desktop morc_menu i3status wmctrl```
 
 # Configuration
-## Create a new XSession
+If you are using Plasma >= 5.25, you need to create a new service to run i3 the login to the Plasma(X11) session. If you are using Plasma < 5.25, you need to create a new XSession and login to that.
```

Comment:
It looks like some text was garbled during editing. Maybe something like "to run i3 instead of kwin upon login the".